packages/hoppscotch-common/src/components/app/Banner.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div
+    class="relative flex items-center px-4 py-2 transition text-tiny group"
+    :class="bannerColor"
+  >
+    <icon-lucide-info class="mr-2 text-white" />
+    <span class="text-secondaryDark">
+      <span v-if="banner.alternateText" class="md:hidden">
+        {{ banner.alternateText }}
+      </span>
+      <span class="<md:hidden">
+        {{ banner.text }}
+      </span>
+    </span>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed } from "vue"
+import { BannerContent, BannerType } from "~/services/banner.service"
+
+const props = defineProps<{
+  banner: BannerContent
+}>()
+
+const bannerColor = computed(() => {
+  const bgColors: Record<BannerType, string> = {
+    error: "bg-red-700",
+    warning: "bg-yellow-700",
+    info: "bg-error",
```

Comment:
How were the background colors chosen? Is `bg-error` the intended color for the `info` variant?

Reply:
According to the PRD, we show red for the error type, yellow for warning and for the info type, I used the previously present colour that was used in the Announcement component. I will now replace bg-error with bg-stone-800. Any suggestions for the type based color scheme are welcome.

Reply:
`bg-stone-800` doesn't go well with `text-secondaryDark`. Also, shouldn't the icon and the text color be the same?

![info](https://github.com/hoppscotch/hoppscotch/assets/25279263/0c2b4c97-8db7-4f6e-b1af-3d7ab2888f53)

Reply:
It will look like the attached screenshot. I will make both the icon and text to `text-secondaryDark`

<img width="1469" alt="image" src="https://github.com/hoppscotch/hoppscotch/assets/70131076/d410b97c-5e63-4dc3-b4d1-dcb6e16527e3">

Reply:
Isn't white the one fitting well in this context?